lib/control-plane/openstackcontrolplane.yaml

```diff
@@ -111,9 +111,9 @@ spec:
       manilaAPI:
         networkAttachments:
           - internalapi
-        replicas: 1
+        replicas: 3
       manilaScheduler:
-        replicas: 1
+        replicas: 3
```

Comment:
I don't like the fact that github does not allow me to comment on something that is not part of this change, but anyway: should L121 of this file be `replicas: 0` ?

Reply:
Per L109 `enabled: false` so the effect is the same.

If we set L121 to 0, then we should also set L114 and L116 to 0 as well and then ensure both service-values.yaml in this PR change update them to 3 1 1 respectively. We will also need additional lines in the kustomization files to actually set it.

Reply:
ok, this explains why we have `replicas: 1` on a `share` that can't be up until the Ceph cluster (or the backend in general) is available.
My comment was mostly about the fact that you might have a topology where manila is enabled `enabled: true` from the very beginning, but, in case the backend (`Ceph` for instance) is deployed in a different stage you need to keep `replicas: 0` until the ctlplane is patched.
My understanding is that everything works because `Manila` is an optional service: it's still `enabled: false` at the ctlplane deployment stage, and we switch the boolean later in time. I don't think we should set `scheduler` and `api` to `0` because they can live w/o a `manilaShare` in any case, and the scope can be reduced to that particular share instance that would crash.
However, if such change generates a problem in other `kustomization.yaml` files that need to be patched as well, I'm ok to not change replicas and reduce the number of changes made by this patch.